README.md

```diff
@@ -50,7 +50,20 @@ and deploy the code.
 
 ### Pre-requisites
 
-1. To run the scripts, Windows users require [Powershell 7.2 (LTS)](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows) or above. Alternatively, you can use a bash terminal using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). macOS users can use a bash terminal.
+1. To run the scripts, Windows users require Powershell 7.2 (LTS) or above. Alternatively, you can use a bash terminal using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). macOS users can use a bash terminal.
+
+1. [Install Git](https://github.com/git-guides/install-git)
+    Run the following to verify that git is available
+    ```ps1
+    git version
+    ```
+
```

Comment:
moved step 3 under step 1. no specific actions yet for mac/bash users so I added a callout that this step is for PowerShell users. Trying to avoid describing that as the windows option as PowerShell 7 is crossplat